### DISCUSSION TITLE
Time Displayed on Dashboard

### DISCUSSION THREAD

**Discussion by SigWar** [Security-Onion-Solutions/securityonion#13523] [Ideas] [answered]
Being an analyst and having to constantly look at the UTC time vs Local time is tedious. Is there a way to display the SO time on the corner (bottom-right) of the dashboard for reference?

**Comment by dougburks** [comment, marked as answer]
Have you seen this part of the documentation from https://docs.securityonion.net/en/2.4/timezones.html?

> Web interfaces like [Alerts](https://docs.securityonion.net/en/2.4/alerts.html#alerts), [Dashboards](https://docs.securityonion.net/en/2.4/dashboards.html#dashboards), [Hunt](https://docs.securityonion.net/en/2.4/hunt.html#hunt), and [Kibana](https://docs.securityonion.net/en/2.4/kibana.html#kibana) should try to detect the timezone of your web browser and then render those UTC timestamps in local time. [Alerts](https://docs.securityonion.net/en/2.4/alerts.html#alerts), [Dashboards](https://docs.securityonion.net/en/2.4/dashboards.html#dashboards), and [Hunt](https://docs.securityonion.net/en/2.4/hunt.html#hunt) allow you to manually set your timezone under Options.
